tests/integration/test_camera_picamera2.py

```diff
@@ -8,6 +8,7 @@
 
 from modules.camera import camera_factory
 
+from modules.camera.camera_config import PiCameraConfig
```

Comment:
No space above this line, and also should be `from modules.camera import camera_config` since we import modules